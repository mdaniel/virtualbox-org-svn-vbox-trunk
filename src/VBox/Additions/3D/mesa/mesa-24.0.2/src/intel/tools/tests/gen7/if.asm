(+f0.0.x) if(8) JIP: LABEL0         UIP: LABEL2                 { align16 1Q };
LABEL0:
(+f0.0) if(8)   JIP: LABEL2         UIP: LABEL1                 { align16 1Q };
(+f0.0) if(8)   JIP: LABEL2         UIP: LABEL1                 { align1 1Q };
LABEL1:
(+f0.0) if(16)  JIP: LABEL2         UIP: LABEL2                 { align1 1H };
(-f0.0) if(8)   JIP: LABEL2         UIP: LABEL2                 { align1 1Q };
(-f0.0) if(16)  JIP: LABEL2         UIP: LABEL2                 { align1 1H };
LABEL2:
