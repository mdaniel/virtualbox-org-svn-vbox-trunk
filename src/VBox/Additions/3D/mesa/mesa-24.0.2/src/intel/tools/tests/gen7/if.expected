22 01 62 00 84 3c 0f 20 04 00 0e 00 02 00 0c 00
22 01 61 00 84 3c 0f 20 04 00 0e 00 0a 00 04 00
22 00 61 00 84 3c 00 20 00 00 00 00 08 00 02 00
22 00 81 00 84 3c 00 20 00 00 00 00 06 00 06 00
22 00 71 00 84 3c 00 20 00 00 00 00 04 00 04 00
22 00 91 00 84 3c 00 20 00 00 00 00 02 00 02 00
